crates/topos-sequencer-subnet-runtime/tests/subnet_contract.rs

```diff
@@ -475,7 +475,7 @@ async fn test_subnet_node_get_block_info(
     )
     .await
     {
-        Ok(mut subnet_client) => match subnet_client.get_next_finalized_block().await {
+        Ok(mut subnet_client) => match subnet_client.get_finalized_block(6).await {
```

Comment:
I see getting `6` here but asserting on `>5` below, why not asserting on `== 6`?

also can you fix the typo (`// ///`) on the comment above this function since you are around